administrator/language/en-GB/plg_schemaorg_blogposting.sys.ini

```
@@ -0,0 +1,7 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_BLOGPOSTING="Schema.org - Blogposting"

```

```suggestion
PLG_SCHEMAORG_BLOGPOSTING="Schema.org - BlogPosting"

```

See https://schema.org/BlogPosting